packages/devtools_app/test/visible_screens_test.dart

```diff
@@ -0,0 +1,183 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/app.dart';
+import 'package:devtools_app/src/code_size/code_size_screen.dart';
+import 'package:devtools_app/src/debugger/debugger_screen.dart';
+import 'package:devtools_app/src/framework_controller.dart';
+import 'package:devtools_app/src/globals.dart';
+import 'package:devtools_app/src/inspector/inspector_screen.dart';
+import 'package:devtools_app/src/logging/logging_screen.dart';
+import 'package:devtools_app/src/memory/memory_screen.dart';
+import 'package:devtools_app/src/network/network_screen.dart';
+import 'package:devtools_app/src/performance/performance_screen.dart';
+import 'package:devtools_app/src/screen.dart';
+import 'package:devtools_app/src/service_manager.dart';
+import 'package:devtools_app/src/timeline/timeline_screen.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'support/mocks.dart';
+
+void main() {
+  group('visible_screens', () {
+    FakeServiceManager fakeServiceManager;
+
+    setUp(() async {
+      fakeServiceManager =
+          FakeServiceManager(useFakeService: true, availableLibraries: []);
+      setGlobal(ServiceConnectionManager, fakeServiceManager);
+      setGlobal(FrameworkController, FrameworkController());
+
+      await serviceManager.isolateManager.selectedIsolateAvailable.future;
+    });
+
+    void setupMockValues({
+      bool web = false,
+      bool flutter = false,
+      bool profileMode = false,
```

Comment:
nit: change to debug mode to match `mockIsDebugFlutterApp`

Reply:
Done!